linux/mcompile/keymap/keymap.cpp

```diff
@@ -12,12 +12,18 @@ int convert_Shiftstate_to_LinuxShiftstate(int shiftState) {
   else if (shiftState == K_SHIFTFLAG)                            return XCB_MOD_MASK_SHIFT;                          // Win ss 16 -> Lin ss 1
   else if (shiftState == (LCTRLFLAG | RALTFLAG))                 return XCB_MOD_MASK_LOCK;                           // Win ss 9  -> Lin ss 2
   else if (shiftState == (K_SHIFTFLAG | LCTRLFLAG | RALTFLAG))   return (XCB_MOD_MASK_SHIFT | XCB_MOD_MASK_LOCK);    // Win ss 25 -> Lin ss 3
-  else                                                           return shiftState;                                  // Lin ss x  -> Lin ss x
+  else return shiftState;                                                                                            // Lin ss x  -> Lin ss x
 }
 
 
-bool ensureValidInputForKeyboardTranslation(int gdk_level, gint count, gint keycode) {
-  if (gdk_level >  (int) count)
+bool ensureValidInputForKeyboardTranslation(int shiftstate, gint count, gint keycode) {
```

Comment:
We no longer need `count`, so should remove it.

Reply:
...done